leakcanary-android-core/src/main/java/leakcanary/internal/AndroidHeapDumper.kt

```diff
@@ -65,12 +65,14 @@ internal class AndroidHeapDumper(
     val toast = waitingForToast.get()
 
     return try {
-      Debug.dumpHprofData(heapDumpFile.absolutePath)
+      val durationMillis = measureTimeMillis {
```

Comment:
measureTimeMillis  is based on System.currentTimeMillis() which isn't recommended for measuring durations (not monotonically increasing).

Reply:
Good point. I have switched to a new util method which calls SystemClock.uptimeMillis()